### PR TITLE
Minor Improvements.

### PR DESCRIPTION
**Pull request by joerg84**
1. Added Null check (instead of size > 0)
2. Generalized supportNamedVips check.

### CODE REVIEW

**Comment by joerg84** [comment]
@nickbp Could you take a look, nothing urgent, TY!

File: src/main/java/org/apache/mesos/offer/OfferAccepter.java [hunk 1]
```diff
@@ -45,7 +46,7 @@ public OfferAccepter(Set<OperationRecorder> recorders) {
   }
 
   public List<OfferID> accept(SchedulerDriver driver, List<OfferRecommendation> recommendations, Filters filters) {
-    if (recommendations.size() <= 0) {
+    if (CollectionUtils.isEmpty(recommendations)) {
```
Comment (nickbp) [inline src/main/java/org/apache/mesos/offer/OfferAccepter.java]: any reason to not just use recommendations.isEmpty()? (same with operations.isEmpty() below)

File: src/test/java/org/apache/mesos/dcos/CapabilitiesTest.java [hunk 2]
```diff
@@ -30,10 +30,18 @@ public void testHasNamedVipsFails() throws URISyntaxException, IOException {
...
 
     @Test
-    public void testHasNamedVipsSucceeds() throws URISyntaxException, IOException {
+    public void testHasNamedVipsSucceeds_1_8() throws URISyntaxException, IOException {
         mockDcosCluster =  MockDcosCluster.create("1.8.0");
         dcosCluster = mockDcosCluster.getDcosCluster();
         Capabilities capabilities = new Capabilities(dcosCluster);
         Assert.assertTrue(capabilities.supportsNamedVips());
     }
+
```
Comment (nickbp) [inline src/test/java/org/apache/mesos/dcos/CapabilitiesTest.java]: Couldn't hurt to add a 2.0 test to exercise that check

Comment (joerg84) [inline src/test/java/org/apache/mesos/dcos/CapabilitiesTest.java]: Sure.

File: src/main/java/org/apache/mesos/offer/OfferAccepter.java (hunk 1 above)
Comment (joerg84) [inline src/main/java/org/apache/mesos/offer/OfferAccepter.java]: CollectionUtils.isEmpty includes a `null` check. If you feel that is not needed here, happy to switch to recommendations.empty().

**Pull request merged**